package.json

```diff
@@ -5,7 +5,7 @@
     "ng": "ng",
     "serve": "ng serve",
     "start": "node server.js",
-    "build": "ng build",
+    "build": "ng build --prod",
```

Comment:
Maybe add a different command for different pipelines? "build": "ng build" for Pull requests and "build:prod": "ng build --prod" for deployment?